src/components/chats/ChatMessage.tsx

```diff
@@ -0,0 +1,82 @@
+import styled, { css } from "styled-components"
+import { Text } from "components/text/Text"
+import { SPACINGS } from "styles/spacings"
+import { useTheme } from "styles/theme/useTheme"
+
+export type ChatMessageProps = {
+  message: string
+  isUserMessage: boolean
+  dateTime: string
+  icon: React.ReactNode
+  color?: string
+}
+
+export const ChatMessage: React.FC<ChatMessageProps> = ({
+  message,
+  isUserMessage,
+  dateTime,
+  icon,
+  color
+}) => {
+
+  const theme = useTheme()
+
+  return (
+    <Container>
+      <MessageContainer
+        isUserMessage={isUserMessage}
+
+      >
+        <MessageText>
+          {message}
+        </MessageText>
+        <IconContainer
+          color={color ?? theme.styles.colorPrimary}
+        >
+          {icon}
+        </IconContainer>
+      </MessageContainer>
+      <Text _css={css`
+        margin-top: 10px;
+        text-align: ${!isUserMessage ? "left" : "right"};
+        color: ${theme.styles.textColorSecondary};
+      `}>
+        {dateTime}
+      </Text>
+    </Container>
+  )
+}
+
+const Container = styled.div`
+  width: 100%;
+  padding: 0px 10px;
+  margin-bottom: 20px;
+`
+
+const MessageContainer = styled.div<{ isUserMessage: boolean }>`
+  width: 100%;
+  display: flex;
+  gap: ${SPACINGS.SM};
+  align-item: flex-start;
+  flex-direction: ${({ isUserMessage }) => !isUserMessage ? "row-reverse" : "row"};
+`
+
+const MessageText = styled(Text)`
+  width: 100%;
+  white-space: pre-wrap;
+  background-color: lightgray;
```

Comment:
This grey doesn't match up with the color used in the designs. However, the color in the designs isnt stored anywhere in our project yet.

We'll need to overhaul the color system soon. But for now, can you create a new variable in the `colors.ts` file called `GREY_200`? Set the value of that variable to `#E9EBED` and use it here please.